scribble-lib/scribble/acmart/acmart.tex

```diff
@@ -4,8 +4,10 @@
 % These are replaced by scribble/acmart/style.tex,
 %  which is used in combination with acmart.cls
 
+\newcommand{\SAuthorinfo}[4]{#1}
```

Comment:
Does this need to take 4 arguments? (And do any of these need to produce output?)